scripts/format.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+#format the source files
+if [ "$#" -eq 0 ]
+then
+	cd ../securedrop
+	yapf -i -r *.py 
+
+	#format unittest files
+	cd ./tests
+	yapf -i -r *.py
```

Comment:
Is this needed? You use the recursive flag above, so test scripts should already be formatted.

Reply:
This isn't resolved.